web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.tsx

```diff
@@ -195,26 +194,24 @@ const BuyCreditsModal: React.FC<BuyCreditsModalProps> = ({
           validate={validationHandler}
           onSubmit={async (values, { setSubmitting }) => {
             setSubmitting(true);
-            if (location.pathname.includes('projects')) {
```

Comment:
the only change here is to remove the `location.pathname` check. previously i thought we only wanted to send the buy2 event from the projects page. this was a definite misunderstanding. we need to send the buy2 event from any page where the buy modal was opened.